conda/recipes/ptxcompiler/meta.yaml

```
@@ -3,6 +3,7 @@
 {% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
 {% set py_version=environ.get('CONDA_PY', 35) %}
 {% set cuda_version='.'.join(environ.get('CUDA', '11.5').split('.')[:2]) %}
```

Should we bump this to 11.6? Guessing we are already doing so in CI

```suggestion
{% set cuda_version='.'.join(environ.get('CUDA', '11.6').split('.')[:2]) %}
```

---

We don't support CUDA 11.6 yet AFAIK